schemas/threat.yml

```diff
@@ -0,0 +1,158 @@
+---
+- name: threat
+  title: Threat
+  group: 2
+  short: Fields describing detected threats and threat intel data
+  description: >
+    ECS threat fields encompass detected suspicious activity (e.g. IDS alert, malware activity) typically
+    reported by IDS/IPS and NGFWs, as well as threat intel/global IOC data ingested from a feed source like MISP or Anomali. 
+    Tactic and technique fields can be populated from information provided by the reporting system, or can be evaluated 
+    based on internal framework preferences at ingest time, or retrospectively.
+
+  type: group
+  fields:
+
+    - name: id
+      level: extended
+      type: keyword
+      short: Reported threat ID.
+      description: >
+        Threat ID, typically provided by reporting system (e.g. NGFW or IDS) or threat intel provider (Anomali, OTX, etc.)
+
+      example: 33227 
+
+    - name: name
+      level: extended
+      type: keyword
+      multi_field:
+        type: text
+        name: text
+      short: Reported threat name.
+      description: >
+        Threat name, typically provided by reporting system (e.g. NGFW or IDS signature) or threat intel provider (Anomali, OTX, etc.)
+
+      example: ntp reserved mode denial of service
+
+    - name: category
+      level: extended
+      type: keyword
+      multi_field:
+        type: text
+        name: text
+      short: Reported threat category.
+      description: >
+        Threat category, typically provided by reporting system (e.g. NGFW or IDS signature) or threat intel provider (Anomali, OTX, etc.)
+
+      example: Denial of Service
+
+    - name: revision
+      format: string
+      level: extended
+      type: long
```

Comment:
Just spotted this. It should be `keyword`. Not all revisions are purely numeric.